internal/pkg/planmanager/plan.go

```diff
@@ -71,16 +72,22 @@ func NewPlan(smgr statemanager.Manager, cfg *configloader.Config) *Plan {
 // All errors will be return.
 func (p *Plan) Execute() []error {
 	errors := make([]error, 0)
-	for _, c := range p.Changes {
+	log.Printf("changes count: %d", len(p.Changes))
```

Comment:
With the recent addition, this file has become larger, reaching 200 lines of code. While it's still very manageable, it would be simpler if we could rearrange them into separate files, which makes it easier to read (less scrolling) and more logical.

Let's do a quick refactor when it's still easy and before it's too late. Some ideas (only examples):

- move structs to a single file, move functions not directly related to plan (for example change, result) in another;
- or, move result related stuff in one file and change related stuff in another

Reply:
Ok, I'll do it right now.

Reply:
commit: 1f1e460cd224734baed21a502c62c562609a7e8a